# HG changeset patch
# User jwe@segfault.lan
# Date 1201899626 18000
#      Fri Feb 01 16:00:26 2008 -0500
# Node ID 22412e3a4641782b0dc45cf256bceb9c3984b154
# Parent  0000000000000000000000000000000000000000
Tailor preparing to convert repo by adding .hgignore

diff --git a/.hgignore b/.hgignore
new file mode 100644
--- /dev/null
+++ b/.hgignore
@@ -0,0 +1,3 @@
+(^|/)CVS($|/)
+(^|/)\.hg($|/)
+(^|/)\.hgtags($|/)
